core/src/test/java/org/apache/accumulo/core/spi/compaction/DefaultCompactionPlannerTest.java

```diff
@@ -586,73 +501,14 @@ public void testErrorOnlyOneMaxSize() {
     EasyMock.expect(senv.getConfiguration()).andReturn(conf).anyTimes();
     EasyMock.replay(conf, senv);
 
-    String executors = getExecutors("'type': 'internal','maxSize':'32M','numThreads':1",
-        "'type': 'internal','numThreads':2", "'type': 'external','group':'q1'");
+    String groups =
+        "[{\"name\":\"small\", \"maxSize\":\"32M\"}, {\"name\":\"medium\"}, {\"name\":\"large\"}]";
     var e = assertThrows(IllegalArgumentException.class,
-        () -> planner.init(getInitParams(senv, executors)), "Failed to throw error");
+        () -> planner.init(getInitParams(senv, groups)), "Failed to throw error");
     assertTrue(e.getMessage().contains("maxSize"), "Error message didn't contain maxSize");
   }
 
-  /**
-   * Tests executors can only have one without a max size.
-   */
-  @Test
-  public void testErrorDuplicateMaxSize() {
```

Comment:
This test was removed because the test comment and exception check statement looked identical to `testErrorOnlyOneMaxSize()`

I'll add the test back and change the check statements to be unique.